aws/resource_aws_subnet_test.go

```diff
@@ -477,7 +509,7 @@ resource "aws_vpc" "test" {
   cidr_block = "10.1.0.0/16"
 
   tags = {
-    Name = "terraform-testacc-subnet"
+  	Name = "terraform-testacc-subnet"
```

Comment:
Nit: The two-space formatting previously here and below is expected:

```suggestion
    Name = "terraform-testacc-subnet"
```